app/chat.py

```
@@ -20,146 +20,140 @@
+
     logger.info(f"Found {len(documents)} files to process")
-    
+
     # Process each file
     all_chunks = []
     detector = ArchetypeDetector()
     chunk_cache = {}  # Cache to store chunks by ID
-    
+
     # First pass: Process all files and collect chunks
     for f, f_hash, f_mtime in documents:
```

Ruff (0.8.2)

Loop control variable `f_hash` not used within loop body

(B007)